exercises/space-age/description.md

```diff
@@ -10,8 +10,7 @@ Given an age in seconds, calculate how old someone would be on:
    - Neptune: orbital period 164.79132 Earth years
 
 So if you were told someone were 1,000,000,000 seconds old, you should
-be able to say that they're 31.69 Earth-years old. Round all ages to
-the nearest hundredth of a year.
+be able to say that they're 31.69 Earth-years old.
```

Comment:
Unlike in #945 the burden of getting the answer properly rounded is shared equally by the maintainers and students via implication.  It is implied that the student will return `31.69 Earth-years`  not 31.689.  But at the same time the maintainers may opt to also be a little easy going and decide to round the students' answers to two decimal places, or otherwise check the answer against a tolerance amount before deciding if it right or wrong.

I think I agree with the assessment that how (and or when) rounding is handled should decided at the track level unless one of the topics envisioned for the exercise is rounding....  So I am in favor of this PR.

Reply:
Whoever writes the test suite is going to have to be comparing two floating-point values. The students do not need to compare; they only need to provide floating-point values to be compared against.

> It is implied that the student will return 31.69 Earth-years not 31.689.

Ah, I see. Would it be helpful to add the word "approximately" to the description, then?

> unless one of the topics envisioned for the exercise is rounding

As I understand it, it is not. I want to find a source to back up my understanding, but it is difficult.

> how (and or when) rounding is handled should decided at the track level

If one of the choices is wrong for some reason or other and therefore should not be shown to students, problem-specifications would not be remiss to specify the correct choice.
My question, then, is whether either choice is wrong. I do not have enough experience with floating-point numbers to answer that question, please help.

Reply:
> > unless one of the topics envisioned for the exercise is rounding
>
> As I understand it, it is not. I want to find a source to back up my understanding, but it is difficult.

Found it. https://github.com/exercism/go/issues/3#issuecomment-45052043

Space Age is meant as a metaprogramming exercise.

Reply:
> Would it be helpful to add the word "approximately" to the description, then?

The feeling I get from this is that the student will not need to do anything further to their answer.

In the exercises that I have worked with that involved floats, I don't think I bothered with rounding.  I was happy if the students answer was within a tolerance (of my design).

I don't think either #945 or #946 is wrong.  I favor #946 because it is more concise.  #945 feels more wishy washy to me.